stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -387,11 +395,11 @@ function getAuthTokenForOAuth2DirectTokenMode(DirectTokenConfig grantTypeConfig,
     }
 }
 
-# Check the validity of the access token which is in the cache. If the expiry time is 0, that means no expiry time is
+# Checks the validity of the access token which is in the cache. If the expiry time is 0, that means no expiry time is
 # returned with the authorization request. This implies that the token is valid forever.
 #
 # + oauth2CacheEntry - OAuth2 cache entry
-# + return - Whether the access token is valid or not
+# + return - `true` if the access token is valid, `false` otherwise
```

Comment:
```suggestion
# + return - `true` if the access token is valid, `false` otherwise
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - `true` if the access token is valid or else `false`
```